lib/Service/PageService.php

```diff
@@ -1101,4 +1110,12 @@ public function getBacklinks(int $collectiveId, int $id, string $userId): array
 
 		return $backlinks;
 	}
+
+	private function generateSlugForPage(string $title, ?File $file): ?string {
+		if (!$file) {
```

Comment:
Can you explain why you check for `$file` here? I don't understand why this would be necessary.

Reply:
I don't remember why. Maybe previously there were cases when `$file` can be nullable, e.g. new page creation.

I've removed this condition and inlined method